main/lsp/LSPTypechecker.h

```diff
@@ -34,9 +34,8 @@ class UndoState;
 class LSPTypechecker final {
     /** Contains the ID of the thread responsible for typechecking. */
     std::thread::id typecheckerThreadId;
-    /** GlobalState used for typechecking. Mutable because typechecking routines, even when not changing the GlobalState
-     * instance, actively consume and replace GlobalState. */
-    mutable std::unique_ptr<core::GlobalState> gs;
```

Comment:
This change is one of those times that I wish `this->` prefixing was pervasive, so it was easier to see at a glance where members were getting mutated versus parameters.